src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -167,3 +173,32 @@ def consume_captions(caption_list: list) -> dict:
             caption_metadata['italicized_bytes'] = utils.create_bytes_to_italicize_text()
 
     return caption_metadata
+
+
+def validate_scene_ids(scene_list: list)
+    """Validates the scene IDs to look for duplicate IDs
+
+    :param scene_list:
+    """
+    scene_ids = {}
+    for scene in scene_list:
+        scene_ids.update(scene['scene_id'] += 1)
```

Comment:
is this line working for you? I get this error trying to run it:
```
 def validate_scene_ids(scene_list: list)
                                           ^
SyntaxError: invalid syntax
```

Reply:
Was missing a colon...